<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-H79G770" failures="2" tests="8" name="Test(failed)" time="0.248" errors="0" timestamp="2024-10-17T08:27:12 IST">
  <testcase classname="org.listeners.FailTests" name="tc1" time="0.044">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase classname="org.listeners.FailTests" name="tc1" time="0.002">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase classname="org.listeners.FailTests" name="tc1" time="0.003">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase classname="org.listeners.FailTests" name="tc1" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.listeners.FailTests.tc1(FailTests.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
  <testcase classname="org.listeners.FailTests" name="tc3" time="0.002">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase classname="org.listeners.FailTests" name="tc3" time="0.002">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase classname="org.listeners.FailTests" name="tc3" time="0.005">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase classname="org.listeners.FailTests" name="tc3" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.listeners.FailTests.tc3(FailTests.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
  <testcase classname="org.listeners.FailTests" name="tc2" time="0.0">
    <ignored/>
  </testcase> <!-- tc2 -->
</testsuite> <!-- Test(failed) -->
